core/node/eth_sender/src/aggregator.rs

```diff
@@ -51,6 +51,57 @@ pub struct Aggregator {
     priority_tree_start_index: Option<usize>,
 }
 
+/// Denotes whether there are any restrictions on sending either
+/// commit, prove or execute operations. If there is one, the reason for it
+/// is stored to be logged.
+#[derive(Debug, Default)]
+pub(crate) struct OperationSkippingRestrictions {
+    pub(crate) commit_restriction: Option<&'static str>,
+    pub(crate) prove_restriction: Option<&'static str>,
+    pub(crate) execute_restriction: Option<&'static str>,
+}
+
+impl OperationSkippingRestrictions {
+    fn skip_and_log(&self, agg_op: &AggregatedOperation, reason: Option<&'static str>) -> bool {
```

Comment:
function name implies it returns true when op should be skipped. Let's either swap true<->false or change name